mods/ra/rules/aircraft.yaml

```diff
@@ -87,7 +87,7 @@ MIG:
 		BuildAtProductionType: Plane
 		BuildPaletteOrder: 50
 		Prerequisites: ~afld, stek, ~techlevel.high
-		BuildDurationModifier: 50
+		BuildDurationModifier: 60
```

Comment:
Is it that "revert build modifiers back to normal for T3 units" change? I suppose it shouldn't be in this PR rather than a separate stuff or a part of a balance PR.

Reply:
oops, wanted to edit BuildPaletteOrder